### ISSUE TITLE
iOS Support

### ISSUE THREAD

**Issue by TomaszWaszczyk** [michaelWuensch/BitBanana#39]
## Description

Would be cool to have the wallet on iOS.

**Comment by michaelWuensch** [comment]
Thanks for your suggestion!

Unfortunately iOS support is not planned.

Here are the reasons why:
- BitBanana is a native Android app. This means the App has to be rewritten from scratch in order to work for iOS.
- Additionally supporting iOS would take a huge amount of time. Time that would be missing to do the Android implementation right. I rather focus on one platform and do it in a proper way than having a half decent thing on multiple platforms.
- I am not familiar with the apple ecosystem and I don't have any Apple deviced needed for depoying and testing.

In short, I am not going to do it, sorry. But BitBanana is open source under MIT License. Whoever wants to create something similar for iOS is welcome to do so.

If you are looking for node management on iOS, try Zeus.

I am keeping this issue open for visibility.

**Comment by michaelWuensch** [comment]
I am keeping this issue open for visibility reason.
